SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriverDebugContext.cpp

```diff
@@ -4,6 +4,7 @@
 #include "EngineClasses/SpatialPackageMapClient.h"
 #include "Interop/Connection/SpatialWorkerConnection.h"
 #include "Interop/SpatialSender.h"
+#include "Interop/SpatialStaticComponentView.h"
```

Comment:
Needed?

Reply:
Yes! Looks like this was being included implicitly somehow through the LB enforcer(?) as when I removed the dependency from there this started breaking. It's definitely used in this file.